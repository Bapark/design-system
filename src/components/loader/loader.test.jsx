import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader.jsx';

describe('Loader', () => {
  it('has defaults', () => {
    const tree = renderer.create((<Loader />)).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // describe('inline API', () => {
  //
  // });

  // describe('when inline is true', () => {
  //   beforeEach(function () {
  //     this.wrapper = this.render({ inline: true });
  //   });
  //
  //   it('applies the correct styles', function () {
  //     expect(this.wrapper.prop('className')).toEqual(styles.inlineWrapper);
  //     expect(this.wrapper.find('span').prop('className')).toEqual(styles.inline);
  //   });
  // });
});
