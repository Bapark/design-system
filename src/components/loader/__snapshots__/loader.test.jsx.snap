// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader has defaults 1`] = `
<div
  className="wrapper"
  data-testid="loader"
>
  <span
    className="spinner"
  >
    Loading...
  </span>
</div>
`;
